src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -63,20 +67,58 @@ def _process_model_before_weight_loading(self, model, **kwargs):
         from compressed_tensors.quantization import apply_quantization_config
 
         ct_quantization_config = self.compressor.quantization_config
-        apply_quantization_config(model, ct_quantization_config, run_compressed=True)
 
-    def _process_model_after_weight_loading(self, model, **kwargs) -> None:
-        pass
+        if self.run_compressed and self.is_quantization_compressed:
+            apply_quantization_config(model, ct_quantization_config, run_compressed=True)
+        elif not self.is_quantization_compressed: 
```

Comment:
this will run for both sparsity cases  and we dont need it to. So we can likely turn it off

Reply:
in models with sparsification, they apply_quantization_config will pass in ct_quant_config, which is None, so it will just return 
https://github.com/neuralmagic/compressed-tensors/blob/2dcbc9d1dd3f4dc29c280efab481b9f0cfde0a27/src/compressed_tensors/quantization/lifecycle/apply.py#L117-L119

Reply:
Right. I guess in what case would we go though apply_quant_config twice?